### PR TITLE
[NUI] VectorGraphics:Open added APIs(DrawableGroup, Picture, Gradient)

### PR DESCRIPTION
**Pull request by JSUYA**
### Description of Change ###

[Add ACR]
src/public/BaseComponents/VectorGraphics/CanvasView.cs
  [Add] public CanvasView()
  [Add] public Size2D ViewBox { get; set; }
  [Add] public void RemoveDrawable(Drawable drawable)
  [Add] public void RemoveAllDrawables()

src/public/BaseComponents/VectorGraphics/Drawable.cs
  [Add] public Vector4 BoundingBox { get; }
  [Add] public void ClipPath(Drawable clip)
  [Add] public void Mask(Drawable mask, MaskType type)

src/public/BaseComponents/VectorGraphics/Shape.cs
  [Add] public Gradient FillGradient { get; set; }
  [Add] public Gradient StrokeGradient { get; set; }
  [Add] public void AddPath(PathCommands pathCommands)

src/public/BaseComponents/VectorGraphics/DrawableGroup.cs
  [Add] public DrawableGroup()
  [Add] public void AddDrawable(Drawable drawable)
  [Add] public void RemoveDrawable(Drawable drawable)
  [Add] public bool RemoveAllDrawables()

src/public/BaseComponents/VectorGraphics/Picture.cs
  [Add] public Picture()
  [Add] public void Load(string url)
  [Add] public void SetSize(Size2D size)
  [Add] public Size2D GetSize()

src/public/BaseComponents/VectorGraphics/Gradient.cs
  [Add] public SpreadType Spread { get; set; }
  [Add] public ReadOnlyCollection ColorStops { get; set; }

src/public/BaseComponents/VectorGraphics/LinearGradient.cs
  [Add] public LinearGradient()
  [Add] public void SetBounds(Position2D firstPoint, Position2D secondPoint)
  [Add] public void GetBounds(ref Position2D firstPoint, ref Position2D secondPoint)

src/public/BaseComponents/VectorGraphics/RadialGradient.cs
  [Add] public RadialGradient()
  [Add] public void SetBounds(Position2D centerPoint, float radius)
  [Add] public void GetBounds(ref Position2D centerPoint, ref float radius)

src/public/BaseComponents/VectorGraphics/ColorStop.cs
  [Add] public ColorStop(float offset, Color color)
  [Add] public float Offset { get; set; }
  [Add] public Color Color { get; set; }

src/public/BaseComponents/VectorGraphics/PathCommands.cs
  [Add] PathCommands(IEnumerable commands, IEnumerable points)
  [Add] public IEnumerable Commands { get; set; }
  [Add] public IEnumerable Points { get; set; }

src/public/BaseComponents/VectorGraphics/VectorGraphicsConstants.cs
  [Add] public enum MaskType { Alpha, AlphaInverse }
  [Add] public enum SpreadType { Pad, Reflect, Repeat }
  [Add] public enum PathCommandType { Close, MoveTo, LineTo, CubicTo }

[Changed ACR]
[Change]
src/public/BaseComponents/VectorGraphics/Shape.cs
public enum VectorGraphics.Shape.FillRuleType { Winding, EvenOdd }
->
src/public/BaseComponents/VectorGraphics/VectorGraphicsConstants.cs
public enum VectorGraphics.FillRuleType { Winding, EvenOdd }

[Change]
src/public/BaseComponents/VectorGraphics/Shape.cs
public enum VectorGraphics.Shape.StrokeCapType { Square, Round, Butt }
->
src/public/BaseComponents/VectorGraphics/VectorGraphicsConstants.cs
public enum VectorGraphics.StrokeCapType { Square, Round, Butt }

[Change]
src/public/BaseComponents/VectorGraphics/Shape.cs
public enum VectorGraphics.Shape.StrokeJoinType { Bevel, Round, Miter }
->
src/public/BaseComponents/VectorGraphics/VectorGraphicsConstants.cs
public enum VectorGraphics.StrokeJoinType { Bevel, Round, Miter }

### API Changes ###

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-446

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/LinearGradient.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ internal LinearGradient(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr
         /// </summary>
         /// <param name="firstPoint">The first point used to determine the gradient bounds.</param>
         /// <param name="secondPoint">The second point used to determine the gradient bounds.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```
Comment (hyunjushin) [inline src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/LinearGradient.cs:52]: not easy for me "a surface constrained by two parallel lines crossing
        /// the given points firstPoint and secondPoint, respectively." ......

File: src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/CanvasView.cs [hunk 2]
```diff
@@ -91,8 +91,11 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
 
         /// <summary>
         /// The viewbox of the CanvasView.
+        /// The viewbox means the size of CanvasView's internal space.
+        /// If the size of the viewbox is larger than the size of the Canvasview, the shapes are displayed smaller than the specified size.
```
Comment (pkosko) [inline src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/CanvasView.cs:95]:
```suggestion
        /// If the size of the viewbox is larger than the size of the CanvasView, the shapes are displayed smaller than the specified size.
```

**Comment by JSUYA** [comment]
@dongsug-song 
https://code.sec.samsung.net/jira/browse/TCSACR-446 is accepted.
Please confirm this PR

**Review by dongsug-song — APPROVED** [review]
ACR accepted

**Pull request merged**